google/cloud/bigquery/retry.py

```diff
@@ -56,3 +56,15 @@ def _should_retry(exc):
 on ``DEFAULT_RETRY``. For example, to change the deadline to 30 seconds,
 pass ``retry=bigquery.DEFAULT_RETRY.with_deadline(30)``.
 """
+
+
+@retry.Retry
```

Comment:
Default deadline is only 2 minutes. I expect we'll want more than that. Especially since `jobs.insert` has a 4 minute timeout on the backend.

Reply:
Do you want to use 4 minutes? 5?

Reply:
Let's do 10 minutes since that was the recommendation the backend folks gave me. Should give us time for at least 2 tries retry, possibly 3.

Reply:
Done